Nickvision.Aura/Keyring/Store.cs

```diff
@@ -259,10 +256,10 @@ public async Task<List<Credential>> GetAllCredentialsAsync()
     public async Task<List<Credential>> LookupCredentialsAsync(string name)
     {
         await _database.OpenAsync();
-        using var cmdQueryCredentials = _database.CreateCommand();
+        await using var cmdQueryCredentials = _database.CreateCommand();
```

Comment:
Why is there await in front of this? Is this for `IAsyncDisposable`?

Reply:
Fleet suggested me this, yes, looks like because of Dispose. And it does the same for every other similar places, just noticed

Reply:
Hmmm i mean I know of the existence of IAsyncDisposable, but I don't think it's really useful compared to regular IDisposable. 

I think it's best to just keep `using var` instead of `await using var`